src/hooks/useEnsureCorrectNetwork.ts

```diff
@@ -62,6 +54,7 @@ export const requestNetworkChange = async (
                     alert('Successfully connected to Polygon Network')
                     return requestAdd
                 } catch (err: any) {
+                    console.warn('02', err)
```

Comment:
Let's not push debug logs out. We may want to write tests for it sometime in the future. Atm it's tech debt, let's say.